relayer/create_ibc_channel.go

```diff
@@ -39,13 +38,13 @@ func (r *Relayer) CreateIBCChannel(logFileOption utils.CommandOption) (Connectio
 		return ConnectionChannels{}, err
 	}
 
-	createConnectionCmd := r.getCreateConnectionCmd()
+	createConnectionCmd := r.getCreateConnectionCmd(override)
```

Comment:
why pass the override flag? you are already checking on it [here](https://github.com/dymensionxyz/roller/pull/312/files#diff-9487a9470d15128b8d2af96afe6b8d4bef406bf7c8609c7f410deb1f6e118f0cR47)

Reply:
it's additional flag on the actual `rly` command